roles/feature-netapp-trident/tasks/post-deploy.yaml

```diff
@@ -0,0 +1,218 @@
+          storage:
+            files:
+            - contents:
+                source: data:text/plain;charset=utf-8;base64,ZGVmYXVsdHMgewogICB1c2VyX2ZyaWVuZGx5X25hbWVzIG5vCiAgIGZpbmRfbXVsdGlwYXRocyBubwogICBlbmFibGVfZm9yZWlnbiAiXiQiCn0KCmJsYWNrbGlzdF9leGNlcHRpb25zIHsKICAgIHByb3BlcnR5ICIoU0NTSV9JREVOVF98SURfV1dOKSIKfQoKYmxhY2tsaXN0IHsKICAgIGRldm5vZGUgIl4ocmFtfHJhd3xsb29wfGZkfG1kfGRtLXxzcnxzY2R8c3QpWzAtOV0qIgogICAgZGV2bm9kZSAiXih0ZHxoYSlkW2Etel0iCiAgICBkZXZpY2UgewogICAgICAgIHZlbmRvciAgIkhQIgogICAgfQogICAgZGV2aWNlIHsKICAgICAgICB2ZW5kb3IgICJJbnRlbCIKICAgIH0KICAgIGRldmljZSB7CiAgICAgICAgdmVuZG9yICAiSEdTVCIKICAgIH0KICAgIGRldmljZSB7CiAgICAgICAgdmVuZG9yICAiREVMTCIKICAgIH0KICAgIGRldmljZSB7CiAgICAgICAgdmVuZG9yICAiSU5URUwiCiAgICB9CiAgICBkZXZpY2UgewogICAgICAgIHZlbmRvciAgIlFFTVUiCiAgICB9CiAgICBkZXZpY2UgewogICAgICAgIHZlbmRvciAgIk5WTUUiCiAgICB9Cn0K
+              filesystem: root
+              mode: 420
+              path: /etc/multipath.conf
+          systemd:
+            units:
+            - name: iscsid.service
+              enabled: true
+            - name: iscsi.service
+              enabled: true
+            - name: multipathd.service
+              enabled: true
+
+- name: Create MachineConfig/90-master-iscsi-multipathing.yaml
+  kubernetes.core.k8s:
+    kubeconfig: "{{ cluster_access_kubeconfig }}"
+    state: present
+    definition:
+      # https://access.redhat.com/articles/7008552
+      # https://docs.netapp.com/us-en/trident/trident-use/worker-node-prep.html#install-the-iscsi-tools
+      apiVersion: machineconfiguration.openshift.io/v1
+      kind: MachineConfig
+      metadata:
+        labels:
+          machineconfiguration.openshift.io/role: master
+        name: 90-master-iscsi-multipathing
+      spec:
+        config:
+          ignition:
+            version: 3.4.0
+          storage:
+            files:
+            - contents:
+                source: data:text/plain;charset=utf-8;base64,ZGVmYXVsdHMgewogICB1c2VyX2ZyaWVuZGx5X25hbWVzIG5vCiAgIGZpbmRfbXVsdGlwYXRocyBubwogICBlbmFibGVfZm9yZWlnbiAiXiQiCn0KCmJsYWNrbGlzdF9leGNlcHRpb25zIHsKICAgIHByb3BlcnR5ICIoU0NTSV9JREVOVF98SURfV1dOKSIKfQoKYmxhY2tsaXN0IHsKICAgIGRldm5vZGUgIl4ocmFtfHJhd3xsb29wfGZkfG1kfGRtLXxzcnxzY2R8c3QpWzAtOV0qIgogICAgZGV2bm9kZSAiXih0ZHxoYSlkW2Etel0iCiAgICBkZXZpY2UgewogICAgICAgIHZlbmRvciAgIkhQIgogICAgfQogICAgZGV2aWNlIHsKICAgICAgICB2ZW5kb3IgICJJbnRlbCIKICAgIH0KICAgIGRldmljZSB7CiAgICAgICAgdmVuZG9yICAiSEdTVCIKICAgIH0KICAgIGRldmljZSB7CiAgICAgICAgdmVuZG9yICAiREVMTCIKICAgIH0KICAgIGRldmljZSB7CiAgICAgICAgdmVuZG9yICAiSU5URUwiCiAgICB9CiAgICBkZXZpY2UgewogICAgICAgIHZlbmRvciAgIlFFTVUiCiAgICB9CiAgICBkZXZpY2UgewogICAgICAgIHZlbmRvciAgIk5WTUUiCiAgICB9Cn0K
+              filesystem: root
+              mode: 420
+              path: /etc/multipath.conf
+          systemd:
+            units:
+            - name: iscsid.service
+              enabled: true
+            - name: iscsi.service
+              enabled: true
+            - name: multipathd.service
+              enabled: true
```

Comment:
On SNO clusters, would this cause a reboot? Then subsequent operations might fail due to the API going down? Maybe wait for the pool to stabilize? Example for that is here:
https://github.com/DanielFroehlich/stormshift-automation/blob/4c216034ef72e5733a1dfabd188f85ac75276a6b/roles/feature-workload-partitioning/tasks/post-deploy.yaml#L32

Reply:
Uf that's not so easy / time consuming. Because If have to patch master, wait... patch worker, wait...